channels/alexa/src/main/kotlin/com/justai/jaicf/channel/alexa/AlexaRequestHandler.kt

```diff
@@ -13,26 +13,34 @@ import com.amazon.ask.model.interfaces.playbackcontroller.PlayCommandIssuedReque
 import com.amazon.ask.model.interfaces.playbackcontroller.PreviousCommandIssuedRequest
 import com.justai.jaicf.api.BotApi
 import com.justai.jaicf.channel.BotChannel
+import com.justai.jaicf.channel.alexa.manager.AlexaBotContextManager
 import com.justai.jaicf.channel.alexa.model.AlexaEvent
 import com.justai.jaicf.channel.http.HttpBotRequest
 import com.justai.jaicf.context.RequestContext
 import java.util.*
 
 class AlexaRequestHandler(
-    override val botApi: BotApi
+    override val botApi: BotApi,
+    useDataStorage: Boolean = false
 ): BotChannel, RequestHandler {
 
+    private val contextManager = useDataStorage.takeIf { it }?.let { AlexaBotContextManager() }
```

Comment:
There is `com.justai.jaicf.helpers.kotlin.ifTrue` function exactly for this purpose

Reply:
It's internal for core module